src/main/java/gridss/cmdline/MultipleSamFileCommandLineProgram.java

```diff
@@ -109,6 +119,57 @@ public List<SAMEvidenceSource> getSamEvidenceSources() {
 	    				getOffset(INPUT_MIN_FRAGMENT_SIZE, i, 0),
 	    				getOffset(INPUT_MAX_FRAGMENT_SIZE, i, 0)));
 	    	}
+			for (int i = 0; i < EMPTY_CATEGORIES.size(); i++) {
```

Comment:
As above - EMPTY is kind of implementation-specific name. Probably better to have something that speaks to the user